lib/logthrdest/logthrdestdrv.c

```diff
@@ -1056,9 +1056,12 @@ _register_stats(LogThreadedDestDriver *self)
 
     _init_stats_key(self, &sc_key);
     stats_register_counter(0, &sc_key, SC_TYPE_DROPPED, &self->dropped_messages);
-    stats_register_counter(0, &sc_key, SC_TYPE_PROCESSED, &self->processed_messages);
     stats_register_counter(0, &sc_key, SC_TYPE_WRITTEN, &self->written_messages);
 
+    stats_cluster_single_key_legacy_set_with_name(&sc_key, self->stats_source | SCS_DESTINATION, self->super.super.id,
```

Comment:
This results in a `StatsClusterKey`, which has a logpipe new-style value and a single legacy value, which is really odd for me, and don't know how should/would behave. Also with this we have two counters, which share the new-style values, but differ in the legacy values, which we do not support. Could you kindly create a separate single legacy key for this, like in `logwriter.c`?

Reply:
All key setter functions sets the full key by zero-initializing it. `stats_cluster_*_key_add_*()` functions can be used to extend a previous set call.

Reply:
Oh yes, this is not an legacy alias adding call, which does not zero init. Thanks for the explanation.